build.gradle.kts

```diff
@@ -133,17 +134,23 @@ val swarmOpenApi: Configuration by configurations.creating {
     }
 }
 
+val swarmOpenApiUpdater: Configuration by configurations.creating {
```

Comment:
Seems to me that a function can be created to avoid duplication

Reply:
Functions in build scripts actually are not the best idea regarding maintenance.
Anyhow, we do not need this 4 times as I wrote in a comment above